application/run-tests.sh

```diff
@@ -1,9 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
Unfortunately, no, because the tests are run by Codeship as a docker container, without the help of Make.